src/main/scala/kinesis/mock/cache/CacheConfig.scala

```diff
@@ -112,16 +112,15 @@ object CacheConfig {
       .traverse {
         case name :: count :: Nil if name.nonEmpty && count.nonEmpty =>
```

Comment:
You'll need to change this so that the `count.nonEmpty` check is not enforced.

Reply:
Thanks for the hint! Totally overlooked the `INITIALIZE_STREAMS` feature (I was mostly relying on the type system / fixing type issues). I also adjusted the README in the last commit.